test-resources/geo/presence-data-schema.json

```diff
@@ -32,6 +32,11 @@
       },
       "additionalProperties": false,
       "required": ["accuracy", "desiredInterval", "minimumDisplacement"]
+    },
+
+    "rawLocations": {
+      "type": ["boolean", "null"],
```

Comment:
I would be interested to understand why we need to support `null` as a valid value here. If I read correctly, this is effectively tri-state - in which case I don't think the `description` adequately captures what that third state conveys.

Reply:
As the `PresenceData` schema is used in both Subscriber and Publisher SDKs we allow sending `null` when the `rawLocations` does not belong to the specific `type` (being subscriber or publisher). Also it makes it easier in terms of backwards compatibility I think :thinking:

Reply:
ok, makes sense. Could you consider adding that explanation to the `description` for this `rawLocations` node?

Reply:
https://github.com/ably/ably-asset-tracking-common/pull/30/commits/aa09c07fcf32a6c42997d5a477509a402c474736